caffe2/ideep/operators/concat_split_op.cc

```diff
@@ -4,6 +4,16 @@
 
 namespace caffe2 {
 
+// Wrap around axis_index if it is negative, s.t., -1 is the last dim
```

Comment:
I'm wondering what's the reason to add a copy here? Operators should be able to directly call [`canonical_axis_index_`](https://github.com/pytorch/pytorch/blob/3408d9de20dc73fc2cd64509d7b4341f76e3bb41/c10/core/TensorImpl.h#L87-L95).